src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -119,24 +119,61 @@ def __init__(self, ws: WorkspaceClient, install_state: InstallState, verify_time
         self._install_state = install_state
         self._verify_timeout = verify_timeout
 
-    def run_workflow(self, step: str, skip_job_wait: bool = False):
+    def run_workflow(self, step: str, skip_job_wait: bool = False, max_wait: timedelta = timedelta(minutes=20)) -> int:
```

Comment:
small nit: let's use `timeout` for the consistency with SDK idioms. also - why do we need `skip_job_wait` if we have waiting timeout? `timedelta(seconds=0)` would mean no wait, right?